modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,246 @@
+| Redelivery Policy | xref:apikit-graphql-module-reference.adoc#redelivery-policy[Redelivery Policy] | Defines a policy for processing the redelivery of the same message. |  | 
+| Body Content | Any | Content of the message (usually, the payload). | #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Array of Any
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config] 
+
+== Types
+
+=== Data Fetcher Attributes
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Alias | String | Alias of the queried field. |  | false
+| Arguments | Object | Arguments of the queried field. |  | false
+| Selected Fields | Object | Map of subfields selected from this compound type. |  | 
+| Source | Any | Actual value of the fetched field parent. |  | false
+|======================
+
+=== Repeatable In Memory Stream
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Initial Buffer Size | Number | This is the amount of memory that is allocated to consume the stream and provide random access to it. If the stream contains more data than buffer capacity, then it is expanded according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  | 
+| Buffer Size Increment | Number | This is by how much buffer size is expanded if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. |  | 
+| Max Buffer Size | Number | This is the maximum amount of memory used. If more is used, a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower or equal to zero means no limit. |  | 
+| Buffer Unit | Enumeration, one of:
+
+* BYTE
+* KB
+* MB
+* GB 
+
+| These attributes are expressed in this unit. |  | 
+|======================
+
+=== Repeatable File Store Stream
+
+|======================
+| Field | Type | Description | Default Value | Required
+| In Memory Size | Number | Defines the maximum memory that the stream should use to keep data in memory. If more is consumed, it starts to buffer the content on disk. |  | 
+| Buffer Unit | Enumeration, one of:
+
+* BYTE
+* KB
+* MB
+* GB 
+
+| The unit in which maxInMemorySize is expressed |  | 
+|======================
+
+=== Redelivery Policy
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Max Redelivery Count | Number | The maximum number of times a message can be redelivered and processed unsuccessfully before triggering a "Process failed" message. |  | 
+| Message Digest Algorithm | String | The secure hashing algorithm to use. If not set, the default is SHA-256. |  | 
+| Message Identifier | * xref:apikit-graphql-module-reference.adoc#redelivery-policy-message-identifier[Redelivery Policy Message Identifier] | Defines which strategy is used to identify the messages. |  | 
```

Comment:
To be consistent with other descriptions:

The strategy to use to identify the messages

Reply:
OK! Thanks!